lsp-headerline.el

```diff
@@ -173,9 +173,17 @@ Switch to current mouse interacting window before doing BODY."
                                      (lsp-headerline--go-to-symbol symbol)))
                                  (define-key map [header-line mouse-2]
                                    (lsp-headerline--make-mouse-handler
-                                     (lsp-headerline--narrow-to-symbol symbol)))
+                                     (-let (((&DocumentSymbol :range (&RangeToPoint :start :end)) symbol))
+                                       (if (= (- (point-max) (point-min)) (- end start))
```

Comment:
Shouldn't you compare point-min with start and point-max with end? If you compare the difference then it will have wrong behaviour if 2 elements have the same size, no?

Reply:
fixed